src/main/java/com/ait/lienzo/client/core/shape/wires/IControlPointsAcceptor.java

```diff
@@ -0,0 +1,63 @@
+package com.ait.lienzo.client.core.shape.wires;
+
+public interface IControlPointsAcceptor
```

Comment:
Great I liked the idea of this Acceptor, in this way we can customize it on Stunner or any other client.